lib/database_connection.rb

```diff
@@ -0,0 +1,11 @@
+require 'pg'
+
```

Comment:
Oh that's your 3rd class! You forgot to mention it in the read me and I was wondering. Very nice. Separation of concerns.